env.js

```diff
@@ -1,13 +1,14 @@
 var log = require('loglevel');
 
 module.exports = function() {
-  if (process.env.COUCH2PG_DEBUG) {
+  if (process.env.COUCH2PG_DEBUG || true) {
```

Comment:
Not sure which bit is dead exactly? You mean just default `loglevel` to output debug mode and if you want to change that change the code?